lib/runner/extensions/http-request.command.js

```diff
@@ -49,6 +49,12 @@ module.exports = {
 
             context = payload.context;
 
+            /**
+             *  points to request currently being processed so that SshTunnelAgent can route the request to
+             *  correct host in case multiple ssh hosts are being configured.
+             */
+            this.currentReq = payload.item.request.url.toString();
```

Comment:
Check if there's a better way to access the current instruction queue from this location.